specification/cognitiveservices/data-plane/ComputerVision/v1.0/ComputerVision.json

```diff
@@ -0,0 +1,1482 @@
+        },
+        "/models/{model}/analyze?overload=url": {
+            "post": {
+                "description": "This operation recognizes content within an image by applying a domain-specific model.  The list of domain-specific models that are supported by the Computer Vision API can be retrieved using the /models GET request.  Currently, the API only provides a single domain-specific model: celebrities. Two input methods are supported -- (1) Uploading an image or (2) specifying an image URL. A successful response will be returned in JSON.  If the request failed, the response will contain an error code and a message to help understand what went wrong.",
+                "operationId": "Recognize Domain Specific Content",
+                "consumes": [
+                    "application/json"
+                ],
+                "produces": [
+                    "application/json"
+                ],
+                "parameters": [
+                    {
+                        "$ref": "../../Common/Parameters.json#/parameters/SubscriptionKey"
+                    },
+                    {
+                        "name": "model",
+                        "in": "path",
+                        "description": "The domain-specific content to recognize.",
+                        "required": true,
+                        "type": "string",
+                        "x-nullable": "false",
+                        "x-ms-enum": {
+                            "name": "DomainModels",
+                            "modelAsString": false
+                        },
+                        "enum": [
+                            "Celebrities",
+                            "Landmarks"
+                        ]
+                    },
+                    {
+                        "$ref": "../../Common/Parameters.json#/parameters/ImageUrl"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "schema": {
+                            "$ref": "#/definitions/DomainModelResults"
+                        }
+                    },
+                    "default": {
+                        "schema": {
+                            "$ref": "#/definitions/ComputerVisionError"
+                        }
+                    }
+                },
+                "x-ms-examples": {
+                    "Successful Domain Model analysis request": {
+                        "$ref": "./examples//SuccessfulDomainModelWithUrl.json"
+                    }
+                }
+            }
+        },
+        "/recognizeText?overload=url": {
+            "post": {
+                "description": "Recognize Text operation. When you use the Recognize Text interface, the response contains a field called “Operation-Location”. The “Operation-Location” field contains the URL that you must use for your Get Handwritten Text Operation Result operation.",
+                "operationId": "RecognizeText",
```

Comment:
`operationId` must be `groupname_methodname` -- 

`groupname` is used to create a class in most generated languages
`methodname` is used to generate a method name in most languages